.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,69 @@
+name: "CI Tests"
```

Comment:
How about naming this file `test.yml`? To me, it seems more standard to name workflow with an action e.g. test, deploy...

Reply:
Updated 954fd18 🎉